benchmark-framework/pom.xml

```diff
@@ -165,6 +165,12 @@
 		</dependency>
 
 		<dependency>
+ 		  <groupId>io.netty</groupId>
+ 		  <artifactId>netty-all</artifactId>
+ 		  <version>4.1.48.Final</version>
```

Comment:
The version should be omitted from here since it's already specified in the parent pom.

Reply:
Done